assembl/static/js/app/views/message.js

```diff
@@ -1534,9 +1534,17 @@ var MessageView = Marionette.LayoutView.extend({
     this.listenToOnce(this.messageModerationOptionsView, 'moderationOptionsSaveAndClose', this.onModerationOptionsSaveAndClose);
     this.listenToOnce(this.messageModerationOptionsView, 'moderationOptionsClose', this.onModerationOptionsClose);
   },
-
+  fetchIdeasCollection:function(){
```

Comment:
This method's contract can be better. () => Promise<ideaCollection>, for example. Having a side-effectful method like this would make us rely on setTimeouts with random time frames...